headers/types/common/common.h

```diff
@@ -897,6 +897,34 @@ struct wan_table {
 };
 ASSERT_SIZE(struct wan_table, 5392);
 
+struct sound_envelope_parameters {
+    uint8_t use_envelope;
+    uint8_t slide_time_multiplier;
+    uint16_t field2_0x2;
+    uint32_t field3_0x4;
```

Comment:
This is actually the new style of default field names in newer versions of Ghidra, so I think either convention is okay. Makes it easier for someone porting notes from a Ghidra project.

Reply:
Interesting. It seems a bit pointless to include the field number.
But if that's how Ghidra generates it, I'm guessing it would be annoying to change it manually for each field.

Reply:
I didn't notice that the other style without the field number was present in this repo. Tbh I prefer not having the field number as well, so I'll change it to that.